openedx/core/djangoapps/debug/views.py

```diff
@@ -23,13 +29,22 @@ def show_reference_template(request, template):
     """
     try:
         uses_bootstrap = u'/bootstrap/' in request.path
-        uses_pattern_library = not uses_bootstrap
+        uses_pattern_library = u'/pattern-library/' in request.path
+        is_legacy = u'/legacy/' in request.path
         context = {
-            "disable_courseware_js": True,
+            "disable_courseware_js": not is_legacy,
             "uses_pattern_library": uses_pattern_library,
             "uses_bootstrap": uses_bootstrap,
         }
         context.update(request.GET.dict())
+
+        # Add some messages to the course skeleton pages
+        if u'course-skeleton.html' in request.path:
+            register_info_message(request, _('This is a test message'))
+            register_success_message(request, _('This is a success message'))
+            register_warning_message(request, _('This is a test warning'))
+            register_error_message(request, _('This is a test error'))
```

Comment:
Should have a test using `HTML('Test message with <b>html</b>.')` and showing that the message is not escaped.